client/src/pages/Navbar.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+
+import { AppBar, Toolbar, InputBase, Paper, Typography, IconButton } from "@material-ui/core";
+import SettingsIcon from '@material-ui/icons/Settings';
+import SearchIcon from '@material-ui/icons/Search';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(theme => ({
+  title: {
+    width: '160px'
+  },
+  titlePart: {
+    color: '#30336b'
+  },
+  search: {
+    width: '50%',
+    height: '90%',
+    borderRadius: '50px',
+    margin: 'auto'
+  },
+  input: {
+    width: 'calc(100% - 58px)',
+    marginLeft: '10px'
+  },
+  settings: {
+    float: 'right'
+  }
+}));
+
+function Navbar(props) {
+  const classes = useStyles();
+  if (props.where === "dashboard") {
```

Comment:
This logic will probably be moved inside the appbar component instead. Semantically might make more sense for this to be a boolean value checking to see if the user is logged in or not, so probably called props.loggedIn or something like that. This would be checked whenever a user makes interactions with components that access protected api routes. (E.g. props.loggedIn ? (return search bar and settings button) : (return display login button) ).

Reply:
Alright that makes sense

Reply:
actually there would be three states, one for the login button, register button, and everywhere else